### DISCUSSION TITLE
Hooks for Allowance in wagmi?

### DISCUSSION THREAD

**Discussion by raffle-x-dev** [wevm/wagmi#3538] [Q&A]
Hi guys.
Sorry for the noob question, but I just started my project in wagmi. How do I manage allowances in wagmi? 
I need to mint my token in change of another erc20 token, so I need my contract to be allowed to spend user's tokens.
Thanks : ))

**Comment by raffle-x-dev** [comment]
Please, can anyone share snippets / examples of allowances in CLI? TY

**Reply by dvncan** [reply]
https://github.com/wevm/wagmi/discussions/1036

**Reply by DobromirKirovLime** [reply]
this thread has linked PR that is closed

**Reply by dvncan** [reply]
still readable tho?

![image](https://github.com/user-attachments/assets/65b164d3-8aa4-4c76-a893-8313a2c02138)
